client/app/pages/dashboards/dashboard.html

```diff
@@ -42,6 +57,7 @@
               </button>
               <ul class="dropdown-menu pull-right" uib-dropdown-menu>
                 <li ng-if="!$ctrl.dashboard.is_archived"><a ng-click="$ctrl.editDashboard()">Edit Dashboard</a></li>
+                <li ng-if="!$ctrl.dashboard.is_archived" ng-class="{disabled: $ctrl.isGridDisabled}"><a ng-click="$ctrl.editLayout(true)">Edit Layout</a></li>
```

Comment:
It seems that the visual cue it gives that it's disabled is the cursor type, but on mobile/tablets there is no cursor... Let's just hide this button when it's not supposed to be available.

Btw, can we use `.visible-md` and `.visible-lg` classes instead of the custom JS logic, no?

Reply:
In Boostrap, by default, that class also changes look off the item. But seems that `superflat_redash` does too much customization :smile: 

About custom JS logic: at this moment - yes, we can use `.visible-*` classes, but it's bad idea for another reason: it's not synchronized with `angular-gridster`. Suppose someone in future will decide to set breakpoint to `900px` - and this menu item will start showing/hiding at a wrong time. So it will look a bug. Therefore it's better to use events provided by `angular-gridster` to know exactly when it's in mobile mode or not. In addition, that event is triggered only when grid mode changes, not on every resize event.

Reply:
👍 

got it. makes sense, so let's just hide it based on the gridster's event.